tests/phpunit/tests/test-switch-blog.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+if ( is_multisite() ) :
+
+	class Switch_Blog_Test extends PLL_Multisites_TestCase {
+
+		/**
+		 * Languages data for their creation keyed by language slug.
+		 * The French language has the same locale as the English one ('en_US').
+		 *
+		 * @var array
+		 */
+		protected $languages = array(
```

Comment:
The critical change is not very visible if we don't read the above comment. I propose 2 solutions to improve this:
1. Don't copy anything and just modify the locale in `set_up()`.
2. Or keep overriding the property and use only 2 languages, let say English US and English India(in this case modify the test string accordingly). 
Moreover the comment could be moved in front of the local to increase the visibility.